Makefile

```
@@ -26,8 +26,25 @@ GO111MODULE=on
 all: test manager addonctl
 
 # Run tests
-test: generate fmt vet manifests
-	go test ./api/... ./controllers/... ./pkg/... ./cmd/... -coverprofile cover.out
+.PHONY: test
+test:
```

Needs the other targets also.
```suggestion
test: generate fmt vet manifests
```